fabric-fluids-v1/src/main/java/net/fabricmc/fabric/api/fluids/containers/FluidContainer.java

```diff
@@ -0,0 +1,57 @@
+package net.fabricmc.fabric.api.fluids.containers;
+
+import java.util.Iterator;
+
+import net.minecraft.util.Identifier;
+
+import net.fabricmc.fabric.api.fluids.containers.volume.FluidVolume;
+import net.fabricmc.fabric.api.fluids.minecraft.FluidIds;
+
+/**
+ * An object that stores fluid, it is assumed that all fluid containers that only hold one fluid fluid Volumes
+ */
+public interface FluidContainer extends Iterable<FluidVolume> {
```

Comment:
FluidVolume extends this class? seems a bit weird.

Reply:
I found it personally convenient to have multicontainers and single containers use the same interface, however I'd be open to better names for the classes